src/Video/ErrorVideoDebug.hpp

```diff
@@ -1,7 +1,20 @@
 #pragma once
 
-#include <Utility/Log.hpp>
 #include <GL/glew.h>
+#include <Utility/Log.hpp>
 
-#define ERROR_CHECK_VIDEO_START {while(glGetError() != GL_NO_ERROR){Log()<<"GL Error\n";}}
-#define ERROR_CHECK_VIDEO_END { GLenum __err(glGetError()); while (__err != GL_NO_ERROR) { Log() << "GL error: " << (const char*)gluErrorString(__err) << "\n"; __err = glGetError(); assert(false); } }
+#define ERROR_CHECK_VIDEO_START               \
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') keyword 'while' not followed by a single space [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT008)